src/util/encoding/packet.py

```diff
@@ -0,0 +1,164 @@
+"""
+Provide a module to decode raw messages (bytes) into structures.
+"""
+import struct
+from dataclasses import dataclass
+
+HEARTBEAT = struct.Struct('<B')
+PROPULSION_MOVEMENT_DONE = struct.Struct('<B')
+PROPULSION_ENCODER_POSITION = struct.Struct('<ii')
+PROPULSION_MOVE_WHEELS = struct.Struct('<ii')
+PROPULSION_STOP = struct.Struct('<')
+SERVO_SET_ANGLE = struct.Struct('<BBB')
```

Comment:
Maybe use a prefix for these "constants" so they are easy to find later? (PACKET_... or something)

Reply:
Also, if this is done, add a prefix or class to the classes below, and stay consistant with it.
Ex: PropulsionEncoderPositionPacket is fine, but PropulsionMovementDone doesn't follow the same rule.

Reply:
Done thanks!